openedx/openedx-ledger
# feat: add Adjustment model and admin

**Description:**
Adds an ``Adjustment`` model, ADR, and admin class for managing adjustment recordds.

**Testing instructions:**
* Find a ledger in need of adjustment, copy the uuid, note the current balance.
* Optionally, find a "transaction of interest" within that ledger to refer to from your adjustment record; copy the uuid.
* Go here http://localhost:18280/admin/openedx_ledger/adjustment/
* Add an adjustment, using the ledger uuid and optionally the transaction of interest uuid from above.
* Fill in the quantity, reason, and notes.
* Save the adjustment.
* Go to the ledger, note that the balance has now changed due to the adjustment.

**Merge checklist:**

**Post merge:**
      finished.
